### PR TITLE
Remove Manual Groups

### PR DESCRIPTION
**Pull request by evantahler**
Remove manual Groups (do not merge until after https://github.com/grouparoo/grouparoo/pull/2681 is merged)

### Development

- [ ] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: pages/docs/config/code-config/groups.mdx [hunk 1]
```diff
@@ -7,13 +7,13 @@ breadcrumb_label: Groups
 
 Once you have an App, Source, and Properties ([start with an App](/docs/running/ui-enterprise/apps) if you don't have any), then you're ready to define your Groups.
 
-Groups are where the magic happens in Grouparoo. (It's in the name, after all!) Groups are a segment or cohort of Records. They are useful in that you can use Groups to target a specific set of users to [export to some Destination](/docs/running/ui-enterprise/destinations). You can have _Calculated Groups_ and _Manual Groups_. [See here](/docs/getting-started/product-concepts) to learn more about the core concepts in Grouparoo.
+Groups are where the magic happens in Grouparoo. (It's in the name, after all!) Groups are a segment or cohort of Records. They are useful in that you can use Groups to target a specific set of users to [export to some Destination](/docs/running/ui-enterprise/destinations). [See here](/docs/getting-started/product-concepts) to learn more about the core concepts in Grouparoo.
```
Comment (andyjih) [inline pages/docs/config/code-config/groups.mdx]:
```suggestion
Groups are where the magic happens in Grouparoo. (It's in the name, after all!) Groups are a segment or cohort of Records. They are useful in that you can use Groups to target a specific set of models to [export to some Destination](/docs/running/ui-enterprise/destinations). [See here](/docs/getting-started/product-concepts) to learn more about the core concepts in Grouparoo.
```

File: pages/docs/config/groups.mdx [hunk 2]
```diff
@@ -12,7 +12,7 @@ pullQuote: "Learn how to configure groups using Grouparoo's UI Config."
   </p>
 </Alert>
 
-Groups are where the magic happens in Grouparoo. (It's in the name, after all!) Groups are a segment or cohort of Records. They are useful in that you can use Groups to target a specific set of users to [export to some Destination](/docs/config/destinations). You can have _Calculated Groups_ and _Manual Groups_. [See here](/docs/getting-started/product-concepts) to learn more about the core concepts in Grouparoo.
+Groups are where the magic happens in Grouparoo. (It's in the name, after all!) Groups are a segment or cohort of Records. They are useful in that you can use Groups to target a specific set of users to [export to some Destination](/docs/config/destinations). [See here](/docs/getting-started/product-concepts) to learn more about the core concepts in Grouparoo.
```
Comment (andyjih) [inline pages/docs/config/groups.mdx]:
```suggestion
Groups are where the magic happens in Grouparoo. (It's in the name, after all!) Groups are a segment or cohort of Records. They are useful in that you can use Groups to target a specific set of models to [export to some Destination](/docs/config/destinations). [See here](/docs/getting-started/product-concepts) to learn more about the core concepts in Grouparoo.
```

**Review by andyjih — CHANGES_REQUESTED** [review]
Just a small tweak, but looks good!

**Review by andyjih — APPROVED** [review]

File: pages/docs/config/code-config/groups.mdx [hunk 3]
```diff
@@ -79,9 +77,9 @@ This Group would contain Records with an email address that is not `@grouparoo.c
 
 More info this below.
 
-## Configuring Your Calculated Group
+## Configuring Your Group
 
-Configuring a Calculated Group is all about adding Rules to the Group. A Rule is some logical operator to Filter Properties. A Record must match every Rule to be included in the Group.
+Configuring a Group is all about adding Rules to the Group. A Rule is some logical operator to Filter Properties. A Record must match every Rule to be included in the Group.
```
Comment (teallarson) [inline pages/docs/config/code-config/groups.mdx:82]: non-blocking nitpick:
```suggestion
Configuring a Group is all about adding Rules to the Group. A Rule is some logical operator to filter Properties. A Record must match every Rule to be included in the Group.
```

**Review by teallarson — APPROVED** [review]
Bye, manual groups!

**Pull request merged**